src/controllers/userController.ts

```diff
@@ -0,0 +1,81 @@
+import { prisma } from '@/model/prisma';
```

Comment:
1. Create a `repositories` folder 
2. Put all DB manipulations there into `userRepository.js`
3. Controllers should only work with requests, responds and call repositories or call services. Services are js files for business logic. So far you don’t have any logic for this folder, so just keep it in mind